src/alert.scss

```diff
@@ -12,7 +12,7 @@ $block: #{$fd-namespace}-alert;
 .#{$block} {
   $fd-alert-border: 1px solid;
   $fd-alert-border-radius: 0.25rem;
-  $fd-alert-padding: 0.5rem 1rem;
+  $fd-alert-padding: 0.375rem 1rem;
```

Comment:
I believe the top and bottom padding should be 0.4375rem. In specs they say 0.5rem (including border)